src/config.d.ts

```diff
@@ -4,6 +4,8 @@ export interface LoadingConfig {
   DISPLAY_PROBABILITIES?: boolean;
   TAGNAME?: string;
   DISPLAY_TITLES?: boolean;
+  MATOMO_HOST?: string;
+  MATOMO_ID?: number;
```

Comment:
You could embed everything under MATOMO,

Like this:

`MATOMO: MatomoConfig | null`;

Then 
`interface MatomoConfig {
host?: string;
id?: number;
}
`

That way you can rely on the object being null, and group the matomo config.

My 2 cents, you can merge as is ;)

Reply:
should be good